lib/node/spvnode.js

```
@@ -274,27 +274,36 @@ class SPVNode extends Node {
     }
   }
 
-  /**
-   * Broadcast a transaction (note that this will _not_ be verified
-   * by the mempool - use with care, lest you get banned from
-   * bitcoind nodes).
-   * @param {TX} tx
-   * @returns {Promise}
-   */
+/**
+ * Add transaction to txFilter, broadcast.
+ * @param {TX} tx
+ */
 
-  sendTX(tx) {
-    return this.broadcast(tx);
+SPVNode.prototype.sendTX = async function sendTX(tx) {
+  try {
```

sendTX does not need to silence errors, they can be only silenced in relay.